### ISSUE TITLE
In Cloud, migration tool reports error if Marvel is enabled in the 2.x cluster

### ISSUE THREAD

**Issue by AlexP-Elastic** [elastic/elasticsearch-migration#86]
See elastic/cloud#2791:

> To reproduce:
> - Create a 2.x cluster (eg 2.4.1)
> - Start shield
> - Make (eg) self monitoring
> - Run the migration tool
> - Get one error:
>   - `marvel.agent.interval** has been renamed to xpack.monitoring.collection.interval`
> - In practice this can be safely ignored and after migration, the setting is renamed
>   - (eg use Sense to examine /_cluster/settings - the old settings have been placed into archive)

I believe this should be handled as part of the `?cloud` param (see #66)

**Comment by AlexP-Elastic** [comment]
(cc @alexbrasetvik)

**Comment by loekvangool** [comment]
I'm now getting these unknown settings which I cannot do anything about as an end-user of Cloud:

```
Cluster Settings
Renamed settingsℹ
marvel.agent.interval has been renamed to xpack.monitoring.collection.interval
Unknown settingsℹ
marvel.agent.exporters.__no-default-local__.enabled will be moved to the archived namespace on upgrade
marvel.agent.exporters.__no-default-local__.type will be moved to the archived namespace on upgrade
marvel.agent.exporters.found-user-defined.enabled will be moved to the archived namespace on upgrade
marvel.agent.exporters.found-user-defined.type will be moved to the archived namespace on upgrade
```

**Comment by clintongormley** [comment]
@alexbrasetvik can you provide me with a list or pattern of further settings which should be ignored on cloud?

**Comment by alexbrasetvik** [comment]
@clintongormley I think `marvel.*` should be good, haven't heard of any other false positives from the migration assistant.

**Comment by clintongormley** [comment]
Fixed in f6c331c87507693ef82ff5c5432c946c4fc1d1b8
